code/modules/admin/verbs/mapping.dm

```diff
@@ -315,6 +315,39 @@ GLOBAL_VAR_INIT(say_disabled, FALSE)
 	for(var/x_number in 1 to 4)
 		final.Insert(icon('icons/hud/screen_gen.dmi', "x[x_number == 1 ? "" : x_number]"), "x[x_number == 1 ? "" : x_number]")
 	fcopy(final, "icons/mob/landmarks.dmi")
+*/
```

Comment:
Properly format the edits, marking the start and the end

Reply:
whoops, forgot to undo this

Reply:
proper start/end markers placed